src/EmbeddedExplorer.ts

```diff
@@ -148,6 +150,16 @@ export class EmbeddedExplorer {
     return `${EMBEDDABLE_EXPLORER_URL}?${queryString}`;
   };
 
+  sendHandshakeToEmbed(message: string) {
```

Comment:
I don't think we need this to be a class function, actually, since its just used in the one place, we can inline it until we need to. 



Also, now that we are adding more post messages outgoing to the embeddable explorer class - can we add all the options as a [type union like we do for the incoming messages](https://github.com/mdg-private/studio-ui/pull/6177/files#diff-e76f66f54fd1d6ed0ab1bde1c0572ac2eb8fa84132bfd834fac5d9c0193d7b63R20) in `studio-ui`? We could copy this exactly without the `MessageEvent`.